chef/api.py

```diff
@@ -80,7 +87,7 @@ def __init__(self, url, key, client, version='0.10.8', headers={}):
         self.key = key
         self.client = client
         self.version = version
-        self.headers = dict((k.lower(), v) for k, v in headers.iteritems())
+        self.headers = dict((k.lower(), v) for k, v in headers.items())
```

Comment:
This would have to use `six.iteritems` to match semantics on Python 2. This is a common issue below so not pointing out each instance.